Home_Task_7.py

```diff
@@ -0,0 +1,49 @@
+"""
+    Task 7: String conversion.
+
+    Description:
+
+Convert a non-negative integer num to its English words representation.
+Example 1:
+Input: num = 123
+Output: "One Hundred Twenty Three"
+
+let's take that number always > 100 and only three digits
+100 <= num <= 999
+"""
+
+ones = {'1': 'One', '2': 'Two', '3': 'Three', '4': 'Four', '5': 'Five',
+        '6': 'Six', '7': 'Seven', '8': 'Eight', '9': 'Nine'}
+dozens = {'2': 'Twenty ', '3': 'Thirty ', '4': 'Forty ', '5': 'Fifty ',
+          '6': 'Sixty ', '7': 'Seventy ', '8': 'Eighty ', '9': 'Ninety '}
+hundreds = {'1': 'One Hundred ', '2': 'Two Hundreds ', '3': 'Three Hundreds ',
```

Comment:
"hundreds" is not usable word - usually we write simple "hundred" . You can check vice-versa translation in google.
So here word hundred could be removed from data